clients/go/valkey-GLIDE.json

```diff
@@ -2,9 +2,10 @@
     "name": "valkey GLIDE",
     "description": "Valkey GLIDE is designed for reliability, optimized performance, and high-availability, for Valkey and Redis OSS based applications. GLIDE is a multi language client library, written in Rust with programming language bindings, such as Java, node.js and Python.",
     "repo":"https://github.com/valkey-io/valkey-glide/tree/main/go",
+    "website": "https://glide.valkey.io/languages/go/",
     "installation": "go get github.com/valkey-io/valkey-glide/go",
-    "version":"v1.3.0",
-    "version_released":"2025-02-15",
+    "version":"v2.1.1",
```

Comment:
What is your release frequency? It seems like this is going to be a lot of work to do if you want to keep the website up-to-date on every patch release.

Have you considered making an action to automate these updates? Probably easier on you and the reviewer.

Reply:
Let me bring this up to the team. Having automation for these sort of things would be great.